scripts/release-sample-data/publish-sample-data.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+TAG_NAME=sampledata
+SAMPLE_DATA_ASSETS_FOLDER=sample-data-assets
+
+echo "-----"
+echo "Downloading LATEST sample data for 5.0"
+
+curl -H "Authorization: token $GHT_PRIVATE_REPO_TOKEN" -L "https://github.tools.sap/cx-commerce/spartacussampledata/archive/$SAMPLE_DATA_NEW.zip" -o "spartacussampledata-current.zip"
```

Comment:
I don't get any error when I use an invalid token or select a branch that doesn't exist. For example 'release/212232/next'. It generates an empty tar/zip file and creates the tag. I think we should produce an error here?

Reply:
checking

Reply:
applied the changes :) 

https://github.com/SAP/spartacus/pull/16134/commits/29a7498e80dbd54ddd73ad7bf90235e87515bd3b